src/gui/application.cpp

```diff
@@ -72,11 +72,15 @@ void migrateConfigFile(const QCoreApplication *app)
 {
     using namespace OCC;
 
+    static const auto logPrefix = QStringLiteral("Settings migration: ");
```

Comment:
Maybe Q_FUNC_INFO?

Reply:
Wouldn't that create a lot more noise (because it includes technical information such as the class, the return type and the arguments) than necessary? We have not done this in the legacy settings migration either, but instead use a user-friendly string.

Reply:
Well the log messages are for the devs to read not the user.
It will also help to locate the origin, I mean you can grep for "Settings migration:" but `migrateConfigFile(const QCoreApplication *app)` will be more clear.